synchro/synchrotest.py

```diff
@@ -203,18 +201,16 @@
     # Tests that use warnings.catch_warnings which don't show up in Motor
     'TestCursor.test_min_max_without_hint',
 
-    # TODO: MOTOR-280
-    'TestTransactionsConvenientAPI.*',
-]
+    # Motor's change streams need Python 3.5 to support async iteration but
+    # these change streams tests spawn threads which don't work without an
+    # IO loop.
+    '*.test_next_blocks',
+    '*.test_aggregate_cursor_blocks',
 
-if sys.version_info[:2] >= (3, 5):
-    excluded_tests.extend([
-        # Motor's change streams need Python 3.5 to support async iteration but
-        # these change streams tests spawn threads which don't work without an
-        # IO loop.
-        '*.test_next_blocks',
-        '*.test_aggregate_cursor_blocks',
-    ])
+    # Can't run these tests because they use threads.
+    '*.test_ignore_stale_connection_errors'
+    '*.test_ignore_stale_connection_errors'
```

Comment:
Duplicate line? Also CI is failing because these lines are missing trailing commas (so these two lines turn into one `'*.test_ignore_stale_connection_errors*.test_ignore_stale_connection_errors'`).